src/sentry/static/sentry/app/components/navTabs.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function NavTabs(props) {
+  let {children, underlined, ...tabProps} = props;
```

Comment:
Seems like `underlined` should default to true -- I would maybe even invert the logic and have a `hideUnderline` prop

Reply:
Hmm I guess it is split close to 50/50

Reply:
Yeah the almost even split made me want to have an opt-in flag vs an opt-out one.